.github/workflows/main.yml

```diff
@@ -115,8 +119,8 @@ jobs:
     - name: Run Tests
       run: |
         mkdir $SCREENSHOT_DIR
+        pytest pyqtgraph/examples -v -n 2
```

Comment:
Shouldn't tests come before examples since the former takes a lot less time than latter? (fast fail)

Reply:
sounds reasonable, order changed 👍🏻